base/sparse/sparsematrix.jl

```diff
@@ -163,7 +163,7 @@ end
 
 ## Reinterpret and Reshape
 
-function reinterpret(::Type{T}, a::SparseMatrixCSC{Tv}) where T where Tv
+function reinterpret(::Type{T}, a::SparseMatrixCSC{Tv}) where {T, Tv}
```

Comment:
Yes :). I went through the commits sequentially, so chances are I commented on a few other things similarly fixed in later commits. Please ignore as appropriate :).